Sources/AGGRenderer/CPPAGGRenderer/CPPAGGRenderer.cpp

```diff
@@ -279,6 +279,35 @@ namespace CPPAGGRenderer{
       agg::render_scanlines(m_ras, m_sl_p8, ren_aa);
     }
 
+    void draw_solid_rect_with_border(const float *x, const float *y, float thickness, float r_fill, float g_fill, float b_fill, float a_fill, float r_stroke, float g_stroke, float b_stroke, float a_stroke){
```

Comment:
You're missing a `const void *object` parameter here, which is why the build is failing. Your header declared that such a function exists, but the linker couldn't find it because this parameter is missing.